nsv13/code/modules/overmap/starmap.dm

```diff
@@ -11,7 +11,7 @@
 
 /obj/machinery/computer/ship/navigation
 	name = "\improper FTL Navigation console"
-	desc = "A computer which can interface with the FTL drive to allow the ship to travel vast distances in space."
+	desc = "A computer which can interface with the Thirring Drive to allow the ship to travel vast distances in space."
```

Comment:
If we call it the Thirring drive making it spin violently before jumping would be funny...

Reply:
Ooo. Like as in making the ship spin? That'd be cool